projects/Glados-PortalNetwork.md

```diff
@@ -0,0 +1,76 @@
+## Motivation
+
+The problems my project is solving is creating useful ways to monitor and visualize the health of the portal network and improving glados in its entirety in every way I can, either that be by writing tests, improving glados ui, creating graphs and improving the speed at which everything loads. This is very important because it allows for more diversity into who’s able to view and visualize the health of glados especially for user’s but is all helpful for the developers as well. It is also very important because glados is the tool that displays the health and insights of the entire Portal Network and the more ways to visualize the health of the Portal Network and where things are going wrong or failing the better. 
+
+
+## Project description
+
+My proposed solution for my problem is to create useful ways to monitor and visualize the health of the portal network and improving glados in its entirety by creating different graphs, writing tests, working on dynamically loading everything so it does not run in linear time and improving the ui. Possibly also lower level improvements depending how things progress and time I have left.
+
+## Specification
+
+- Graphs: Create a list of interesting data that would be useful and helpful to glados if it was displayed on the page, then figure how that information would be best suited to be display on glados by making different graphs best suited for the information i'm displaying, then figure out how to pull and display the information I want to display through the graphs using sea-orm or sea-query. If these graphs are useful and insightful toward glados and the team approves of them they might be implemented
```

Comment:
It would be useful for you to talk about the different graphs you might create. What sort of data are you looking to visualize? You talk about being able to observe the health of the network. What does that entail?